src/views/Weather-Components/Header.js

```diff
@@ -1,29 +1,78 @@
-import React from "react";
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
 import TextField from "@mui/material/TextField";
-import { Cities } from "./Constant";
 import Autocomplete from "@mui/material/Autocomplete";
+
+import { useGetCityAutoCompleteQuery } from "../../services/WeatherService";
 export const Header = () => {
+  const navigate = useNavigate();
+  const handleRedirect = (id) => {

```

Comment:
Try to keep local functions below react hooks.

Reply:
ok